recipes-bsp/firmware-imx/firmware-nxp-wifi_1.0.bb

```diff
@@ -194,3 +194,5 @@ RDEPENDS:${PN}-nxpiw612-sdio += "${PN}-nxp-common"
 RPROVIDES:${PN}-nxpiw612-sdio = "linux-firmware-nxpiw612-sdio"
 RREPLACES:${PN}-nxpiw612-sdio = "linux-firmware-nxpiw612-sdio"
 RCONFLICTS:${PN}-nxpiw612-sdio = "linux-firmware-nxpiw612-sdio"
+
+COMPATIBLE_MACHINE = "(imx-nxp-bsp)"
```

Comment:
Is it compatible with `imx-mainline-bsp`?

Reply:
yes, it is also compatible with `imx-mainline-bsp` but imo for mainline is better to use version provided by `linux-firmware`

Reply:
So shouldn't it be marked compatible but not recommended for mainline?